doc/en/developer/source/eclipse-guide/index.rst

```diff
@@ -18,6 +18,16 @@ Running and debugging
 Run or debug the class ``org.geoserver.web.Start`` in the ``web-app`` 
 module. The steps to do so are detailed in the :ref:`quickstart`.
 
+Running GeoServer with Extensions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, GeoServer will run without any extensions enabled. In order to run GeoServer with extensions, the ``web-app`` module declares a number of profiles used to enable specific extensions when running ``Start``. To enable an extension, re-generate the ``web-app`` eclipse profile with the appropriate maven profile(s) enabled::
+
+  % cd web/app
+  % mvn eclipse:eclipse -P wps
```

Comment:
Hum, this is a sub-par way imho, as it does not link the project with the wps project, but only with the jars.
Better to stay in the root directory, and do the same command, it will create the eclipse project for the wps module too and link the two project by project, so that you can work directly against the sources and make fixes during the debug session